NitroxClient/MonoBehaviours/PlayerStatsBroadcaster.cs

```diff
@@ -1,40 +1,39 @@
-﻿using NitroxClient.GameLogic;
-using NitroxModel.Core;
+using NitroxClient.GameLogic;
 using UnityEngine;
 
-namespace NitroxClient.MonoBehaviours
+namespace NitroxClient.MonoBehaviours;
+
+public class PlayerStatsBroadcaster : MonoBehaviour
 {
-    public class PlayerStatsBroadcaster : MonoBehaviour
+    private float time;
+    private const float BROADCAST_INTERVAL = 3f;
+    private LocalPlayer localPlayer;
+    private Survival survival;
+
+    public void Awake()
     {
-        private float time;
-        private const float INTERPOLATION_PERIOD = 4.00f;
-        private LocalPlayer localPlayer;
+        localPlayer = this.Resolve<LocalPlayer>();
+        survival = Player.main.GetComponent<Survival>();
+    }
 
-        public void Awake()
-        {
-            localPlayer = NitroxServiceLocator.LocateService<LocalPlayer>();
-        }
+    public void Update()
+    {
+        time += Time.deltaTime;
 
-        public void Update()
+        // Only do on a specific cadence to avoid hammering server
+        if (time >= BROADCAST_INTERVAL)
         {
-            time += Time.deltaTime;
+            time = 0;
 
-            // Only do on a specific cadence to avoid hammering server
-            if (time >= INTERPOLATION_PERIOD)
+            if (survival)
```

Comment:
Can this be checked in awake and just remove this component instead of checking in Update and doing nothing?